specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -6935,7 +6935,11 @@
         "allowExtensionOperations": {
           "type": "boolean",
           "description": "Specifies whether extension operations should be allowed on the virtual machine. <br><br>This may only be set to False when no extensions are present on the virtual machine."
-        }
+        },
```

Comment:
adding new property to existing version is breaking change. suggest to add in new service tag. definition of breaking change https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines

Reply:
@yungezz this is coming across the wire already (see the response above) and thus presumably defaulted - so this is just missing from the Swagger definition?

Reply:
taking a look into this it appears that scaleInPolicy is supported in the API but missing from the Swagger too: https://docs.microsoft.com/en-us/rest/api/compute/virtualmachinescalesets/createorupdate

Reply:
also it appears the repository being linked to isn't public?